.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Explanation
+
+<!--
+In a couple sentences, explain why this PR is needed and what it addresses. This should be an explanation a non-developer user can understand and covers the "why" question. It should also clearly indicate whether this PR represents an addition, a change, or a fix of existing behavior. This explanation will be used to assist in the release note drafting process.
+
+THIS IS MANDATORY.
+-->
+
+## Related issue
+
+<!--
+Please link the GitHub issue this pull request resolves in the format of `Closes #1234`.
```

Comment:
@octonawish-akcodes We can also include Fixes keyword, like, `Fixes #1234`

Reply:
done